nautobot_ssot/filters.py

```diff
@@ -38,6 +38,12 @@ class SyncLogEntryFilterSet(BaseFilterSet):  # pylint: disable=too-many-ancestor
         }
     )
 
+    sync = NaturalKeyOrPKMultipleChoiceFilter(
+        to_field_name="name",
+        queryset=models.Sync.objects.all(),
+        label="Sync (name or ID)",
+    )
```

Comment:
There's no `name` field on `Sync`. I don't think we want a `NaturalKeyOrPKMultipleChoiceFilter`. This should be a `ModelMultipleChoiceFilter` I think.